habitat-lab/habitat/articulated_agent_controllers/humanoid_rearrange_controller.py

```diff
@@ -152,8 +152,9 @@ def calculate_walk_pose(
         :param distance_multiplier: allows to create walk motion while not translating, good for turning
         """
         deg_per_rads = 180.0 / np.pi
+
         forward_V = target_position
-        if forward_V.length() < EPS:
+        if forward_V.length() < EPS or np.isnan(target_position).any():
```

Comment:
Should we instead raise an exception if target_position is NaN?

Reply:
Unfortunately this frequently happens in my code. So raising an exception would crash the run. We should figure out what the long term fix is.